### ISSUE TITLE
Hard coded file path

### ISSUE THREAD

**Issue by ConnorJWalker** [TSEG24/gamemotion#1]
Line 18 in GameLogic.py has a hardcoded file path. This will make it so it will only ever work on one computer. This cannot happen in our final project - this *will* create problems if this is ever distributed - create a relative filepath. (this is important for every language)

We are better off forcing the user to have the two programs installed in a certain way so we can use a relative filepath instead of using a hardcoded file. I can even create a file to force this. Please, we can *never* include a hardcoded filepath. Doing so will also make it so only the person with this filepath (the original/source programmer) can test before pushing. This is a huge issue.

**Comment by NoahChar** [comment]
Issue not completely resolved, some file paths still hardcoded.
Please resolve ASAP.

**Comment by Progr4m4573R** [comment]
Please provide more detail as to which file paths are hardcoded.
I have explicitly said that the vision team will need to modify the Open CV part of the code which i fear you are refering to, the code will run, provided your part does. 

I have mearly written my own c++ code to copy what the openCV code would do but i have not uploaded this in the code as it is redundant since you have the OpenCV code.

Stephen

**Comment by ConnorJWalker** [comment]
Line 16 and 17 are hardcoded. This can be hard to get around becasue we can't know where the file paths are going to be installed. My thoughts to get around this is for us to create an installing shell script that will store both the vision and robot project in the same directory. If we do this then we will know the file paths that will be needed as the installer will always put them in the same place.

**Comment by Progr4m4573R** [comment]
Yes 16 and 17 are the only things to do with OpenCv that i have had to make my own code for.

Stephen
